pom.xml

```diff
@@ -175,4 +190,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>  
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <configuration>
+                    <compatibleSinceVersion>2.0</compatibleSinceVersion>
```

Comment:
https://github.com/jenkinsci/maven-hpi-plugin/blob/b3052ae8d3204b0f6630c94c4d7ed930443f6007/src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractHpiMojo.java#L114-L117

> the oldest version of this plugin which the current version is configuration-compatible with

If we add new additional fields (the majority of the time changes in plugins are this category), on downgrade the new fields will be silently ignored

If we migrate fields - which is what we are doing here - then the old field will no longer be available after the job has been saved, which is exactly a break in configuration compatibility, which is why this is the marker in the sand